packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -346,12 +346,15 @@ class _ListTitleItem extends StatelessWidget {
   final Widget? leading;
 
   @override
-  Widget build(BuildContext context) => Card(
+  Widget build(BuildContext context) => SmoothCard(
         child: ListTile(
           onTap: onTap,
           title: Text(title),
           subtitle: subtitle == null ? null : Text(subtitle!),
-          leading: leading ?? const Icon(Icons.edit),
+          leading: Column(
```

Comment:
I cannot see the point: all icons are supposed to have the same width - and they do. And if they don't, they should.

Reply:
It's not about the width. It's about the height of the container that varies because of the different text size. With a Column icons are centered vertically. Otherwise they will remain on the top.

Reply:
I understand. There's probably another way to do it but OK with the `Column`. But please add a comment like `// we use a Column to have the icon centered vertically`

Reply:
Sure!